code/modules/power/cable.dm

```diff
@@ -107,12 +107,20 @@ By design, d1 is the smallest direction and d2 is the highest
 	if(powernet)
 		cut_cable_from_powernet()				// update the powernets
 	GLOB.cable_list -= src							//remove it from global cable list
+
+	//If we have a stored item at this point, lets just delete it, since that should be
+	//handled by deconstruction
+	if(stored)
+		QDEL_NULL(stored)
 	return ..()									// then go ahead and delete the cable
 
 /obj/structure/cable/deconstruct(disassembled = TRUE)
 	if(!(flags_1 & NODECONSTRUCT_1))
-		var/turf/T = loc
-		stored.forceMove(T)
+		var/turf/T = get_turf(loc)
+		if(T)
+			stored.forceMove(T)

```

Comment:
if this doesn't null `stored`, won't it still be deleted in `Destroy`?

Reply:
^

Reply:
good point, I'll add that, nice catch